src/components/Search/SearchOverTime.tsx

```diff
@@ -17,38 +18,69 @@ import {
   SearchStyles,
 } from '../../styles';
 import { Changeset, Repository } from '../../types/state';
+import { TimeRange } from '../../types/types';
 import SearchDropdown from './SearchDropdown';
 import SearchInputAndResults from './SearchInputAndResults';
 import SelectedRevisions from './SelectedRevisions';
+import TimeRangeDropdown from './TimeRangeDropdown';
 
 interface SearchProps {
-  selectLabel: string;
-  tooltip: string;
-  inputPlaceholder: string;
   hasEditButton: boolean;
   displayedRevisions: Changeset[];
-  repository: Repository['name'];
+  baseRepo: Repository['name'];
+  newRepo: Repository['name'];
+  timeRangeValue: TimeRange['value'];
   onRemoveRevision: (item: Changeset) => void;
   onSearchResultsToggle: (item: Changeset) => void;
-  onRepositoryChange: (repo: Repository['name']) => unknown;
+  onBaseRepositoryChange: (repo: Repository['name']) => unknown;
+  onNewRepositoryChange: (repo: Repository['name']) => unknown;
+  onTimeRangeChange: (val: TimeRange['value']) => unknown;
 }
 
 export default function SearchOverTime({
-  selectLabel,
-  tooltip,
-  inputPlaceholder,
   hasEditButton,
-  repository,
+  baseRepo,
+  newRepo,
   displayedRevisions,
+  timeRangeValue,
   onRemoveRevision,
   onSearchResultsToggle,
-  onRepositoryChange,
+  onBaseRepositoryChange,
+  onNewRepositoryChange,
+  onTimeRangeChange,
 }: SearchProps) {
   const mode = useAppSelector((state) => state.theme.mode);
   const styles = SearchStyles(mode);
+  const timeRangeStrings =
+    Strings.components.searchDefault.overTime.collapsed.timeRange;
+  const stringsBase =
+    Strings.components.searchDefault.overTime.collapsed.baseRepo;
+  const stringsNew =
+    Strings.components.searchDefault.overTime.collapsed.revisions;
 
   const [formIsDisplayed, setFormIsDisplayed] = useState(!hasEditButton);
 
+  const baseRepoStyles = style({
+    marginBottom: `${Spacing.Large}px`,
+    justifyContent: 'space-between',
+    $nest: {
+      ' > div': {
+        flexGrow: 1,
```

Comment:
My understanding is that you can use `<Grid item xs>` to make an item in the grid automatically grow.
See https://mui.com/material-ui/react-grid/#auto-layout

Reply:
Thanks for mentioning this. Sometimes I forget MUI has these things covered. I'll look to stick with MUI in future patches and override unless absolutely necessary.